danielMunteanu/src/main/java/danielCollectionTask/Item.java

```diff
@@ -0,0 +1,44 @@
+package danielCollectionTask;
+
+public class Item {
+    final long id;
```

Comment:
add access modifiers...Please, note that FINAL is not an access modifier, it is a class modifier

Reply:
done